cfme/automate/simulation.py

```diff
@@ -46,6 +47,11 @@ def simulate(
         'avp': attributes_values,
     })
     view.submit_button.click()
+    wait_for(
+        lambda: view.flash.is_displayed,
+        delay=10,
```

Comment:
Can we lower this delay please? Any reason to be waiting 10s between each check? I'd check every second